config/params/fork_test.go

```diff
@@ -132,8 +132,8 @@ func TestNextForkData(t *testing.T) {
 		{
 			name:              "after last bpo - should be far future epoch and 0x00000000",
```

Comment:
This test doesn't match the data here. The data is neither far future epoch or 0x000000